examples/campaign_management/update_campaign_criterion_bid_modifier.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sets a bid modifier for the mobile platform on the given campaign."""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.api_core import protobuf_helpers
+
+
+def main(client, customer_id, criterion_id, bid_modifier):
+    campaign_criterion_service = client.get_service(
+        'CampaignCriterionService', version='v2')
+
+    criterion_rname = campaign_criterion_service.campaign_criteria_path(
+        customer_id, criterion_id)
+
+    campaign_criterion_operation = client.get_type(
+        'CampaignCriterionOperation', version='v2')
+    campaign_criterion = campaign_criterion_operation.update
+    campaign_criterion.resource_name = criterion_rname
+    campaign_criterion.bid_modifier.value = bid_modifier
+    campaign_criterion.device.type = client.get_type(
```

Comment:
I actually don't get why we want to set the device here too.
The device would be determined by the specified campaign criterion ID already?
Setting this to different thing than what's specified in the criterion ID would raise an error.

Reply:
Just confirmed that setting device here is a no-op.
What's to be updated is determined by the resource name already.
If we use 30001 (MOBILE) as a criterion ID, and set this to DESKTOP (whose ID is 30000), it still updates only MOBILE, and no error is thrown.

Need to remove this line to prevent confusion.